Nickvision.Aura/Network/INetworkMonitor.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Tmds.DBus;
+
+namespace Nickvision.Aura.Network;
+
+[DBusInterface("org.freedesktop.portal.NetworkMonitor")]
+internal interface INetworkMonitor : IDBusObject
+{
+    Task<bool> GetAvailableAsync();
+    Task<bool> GetMeteredAsync();
+    Task<uint> GetConnectivityAsync();
+    Task<IDictionary<string, object>> GetStatusAsync();
+    Task<bool> CanReachAsync(string Hostname, uint Port);
+    Task<IDisposable> WatchchangedAsync(Action handler);

```

Comment:
Should this be `WatchchangedAsync`? Or `WatchChangedAsync`?

Reply:
The signal name is `changed` (lowercase), so generated method is `WatchchangedAsync` (I know, doesn't look right)
https://flatpak.github.io/xdg-desktop-portal/#gdbus-signal-org-freedesktop-portal-NetworkMonitor.changed

Reply:
I just tried to make it `WatchChangedAsync` and it stopped working.